Send messages to #ToonHQ & pings user

RickyTwitch opened the pull request:
Added a feature which allowed the commands to be ran anywhere in the discord, however, the request command will always go to ToonHQ unless it's a moderation command

Commit:
Send messages to #ToonHQ & pings user

Hardcoded a feature which allowed the commands to be ran anywhere in the discord, however, the request command will always go to ToonHQ unless it's a moderation command

judge2020 commented on handlers/command_handler.py:
```diff
@@ -54,8 +55,9 @@ def _delete_first_two_words(self, phrase: str) -> str:
-        await self.client.send_message(message.channel, ip_help)
+""".format(message)
+
+        await self.client.send_message(discord.Object(id='347411900864135189'), ip_help)
```
What is ID 347411900864135189? If you're trying to send a DM instead of replying in the channel, you can 
```python
await self.client.send_message(message.author, str)
```

RickyTwitch replied:
ID 347411900864135189 is the ToonHQ channel on the discord. I've changed this to ask for a variable instead, which can be found at the top of the script.

judge2020 commented on handlers/command_handler.py:
```diff
@@ -41,6 +41,7 @@ def _delete_first_two_words(self, phrase: str) -> str:
 
     async def command_ip(self, message):
         ip_help = """
+{0.author.mention}
```
Due to a specific discord/discord.py bug, please use
```
<@{0.author.id}>
```
instead.

RickyTwitch replied:
Noted.

judge2020 commented on handlers/command_handler.py:
```diff
@@ -113,6 +115,7 @@ def _delete_first_two_words(self, phrase: str) -> str:
         await self.client.send_message(message.channel, reason)
 
     async def command_status(self, message):
+        mention_user = "{0.author.mention}".format(message)
```
When defining a string with a variable, you can just put the `message.author.mention` part instead of having to format the string

Also, please instead do `<@{id}>` (see above)

RickyTwitch replied:
So instead of the code being:
```
mention_user = "{0.author.mention}".format(message)
```
it should change to:
```
mention_user = "<@{0.author.id}>"
```

judge2020 replied:
you will need `"<@{0.author.id}>".format(message)` since you have other characters in the string

judge2020 commented on handlers/command_handler.py:
```diff
@@ -142,4 +145,5 @@ def _delete_first_two_words(self, phrase: str) -> str:
             4: discord.Colour.dark_red()
         }[worst_status]
-        await self.client.send_message(message.channel, embed=embed)
+        await self.client.send_message(discord.Object(id='347411900864135189'), mention_user)
```
You might be misunderstanding send_message. 
send_message(destination, contents)

The old statement should suffice unless you want to PM the status to the user (see above for PMing the user)

RickyTwitch replied:
Like above `discord.Object(id='Channel ID')` sends a message to the channel stated in the parentheses. DM'ing is also an option, however, keeping the messages to a channel in the discord allows users to run commands on behalf of others that don't know they exist.

judge2020 reviewed (changes requested):
Looks good, glad you're getting into this. The requested changes should just be to get you used to discord.py in the future and understand it better.

Commit:
Updated code Send messages to #ToonHQ & pings user

Fixed issues:
- Added a var so channel ID's aren't all hard coded
- Chnaged mention code

Commit:
Updated code Send messages to #ToonHQ & pings user

Commit:
Revert "Updated code Send messages to #ToonHQ & pings user"

This reverts commit 0a67e6c795ce5670d20831ce54b90b2f4e05b769.
